salt/reviewer-suggestions/init.sls

```diff
@@ -141,9 +141,17 @@ reviewer-suggestions-cron:
         - minute: 0
         - user: {{ pillar.elife.deploy_user.username }}
 
-reviewer-suggestions-server-service-started:
+reviewer-suggestions-server-service-enabled:
     service.running:
-        - order: last
         - name: reviewer-suggestions-server
         - require:
             - file: reviewer-suggestions-server-service
+
+reviewer-suggestions-server-service-started:
+    cmd.run:
+        - order: last
+        - user: {{ pillar.elife.deploy_user.username }}
```

Comment:
also you may not need to run it as the `elife` user: root is the default but it's fine unless we are creating files or directories that get a wrong ownership